themes/src/main/resources-community/theme/base/email/messages/messages_cs.properties

```diff
@@ -1,7 +1,7 @@
 # encoding: utf-8
 emailVerificationSubject=Ověření e-mailu
-emailVerificationBody=Někdo vytvořil účet {2} s touto e-mailovou adresou. Pokud jste to vy, klikněte na níže uvedený odkaz a ověřte svou e-mailovou adresu \n\n{0}\n\nTento odkaz vyprší za {1} minuty.\n\nPokud jste tento účet nevytvořili, tuto zprávu ignorujte.
-emailVerificationBodyHtml=<p>Někdo vytvořil účet {2} s touto e-mailovou adresou. Pokud jste to vy, klikněte na níže uvedený odkaz a ověřte svou e-mailovou adresu. </p><p><a href="{0}">Odkaz na ověření e-mailové adresy</a></p><p>Platnost odkazu vyprší za {1} minut.</p><p>Pokud jste tento účet nevytvořili, tuto zprávu ignorujte.</p>
+emailVerificationBody=Někdo vytvořil účet {2} s touto e-mailovou adresou. Pokud jste to vy, klikněte na níže uvedený odkaz a ověřte svou e-mailovou adresu \n\n{0}\n\nTento odkaz vyprší za {4}.\n\nPokud jste tento účet nevytvořili, tuto zprávu ignorujte.
```

Comment:
Why 4th parameter ({4})? When I'm looking at how the `emailVerificationBody` message is used in `email-verification.ftl` `msg("emailVerificationBody",link, linkExpiration, realmName, linkExpirationFormatter(linkExpiration))` and I compare it with `en` translation:

 "Someone has created a {2} account with this email address. If this was you, click the link below to verify your email address\n\n{0}\n\nThis link will expire within {3}.\n\nIf you didn''t create this account, just ignore this message."

It seems that parameters are indexed from 0 (`link` in this case) which means 4th parameter (`linkExpirationFormatter(linkExpiration)`) will have index 3. This seems to be the case for other translations as well. Am I missing something here?